Dockerfile

```diff
@@ -80,7 +80,8 @@ RUN chmod a+x /var/lib/postgresql/src/generate_tiles.py
 RUN git clone https://github.com/jacobtoye/osm-bright.git /style --depth 1
 COPY etc/configure.py /style/configure.py
 COPY etc/osm-smartrak.osm2pgsql.mml /style/themes/osm-smartrak/osm-smartrak.osm2pgsql.mml
-COPY etc/skate-style/img/shields /style/themes/osm-smartrak/img/shields
+COPY etc/skate-style/img/. /style/themes/osm-smartrak/img

```

Comment:
**question:** is the `/.` at the end of the origin path necessary?

Reply:
Ah thanks, not necessary! I was confusing the [cp docs](https://docs.docker.com/engine/reference/commandline/cp/) for the [COPY docs](https://docs.docker.com/engine/reference/builder/#copy)